main.py

```diff
@@ -9,7 +9,7 @@
 import scipy.optimize as opt
 import scipy.stats as stat
 
-import constants
+import config
```

Comment:
Again inconsistent with the `from config import CONST` which is used in other files.

Reply:
Not clearly, this time.